bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -649,6 +653,8 @@ spec:
                   value: quay.io/netobserv/flowlogs-pipeline:v0.1.8
                 - name: RELATED_IMAGE_CONSOLE_PLUGIN
                   value: quay.io/netobserv/network-observability-console-plugin:v0.1.9
+                - name: DOWNSTREAM_DEPLOYMENT
```

Comment:
would `DOWNSTREAM_DEPLOYMENT ` be set to true in CPAAS pipeline stages?

Reply:
Yes, the last step for this task is to update downstream build to change DOWNSTREAM_DEPLOYMENT
We already do this kind of changes for container images such as RELATED_IMAGE_CONSOLE_PLUGIN

Reply:
got it, thanks!. Could you open a PR for downstream setting this env to true? 

we'd have to test this PR post-merge cc @jotak